src/script/lua_api/l_util.cpp

```diff
@@ -245,6 +246,35 @@ int ModApiUtil::l_get_hit_params(lua_State *L)
 	return 1;
 }
 
+// check_password_entry(name, entry, password)
+int ModApiUtil::l_check_password_entry(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	std::string name = luaL_checkstring(L, 1);
+	std::string entry = luaL_checkstring(L, 2);
+	std::string password = luaL_checkstring(L, 3);
+
+
+	if (base64_is_valid(entry)) {
+		std::string gen_entry = translate_password(name, password);
```

Comment:
Could you possibly come up with worse variable names?
How about something like... "hashed_password"?